frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -29,6 +29,9 @@ export const normalizeClusterServiceVersions = (
         }`
       : operatorLogo;
 
+  const formatTileDescription = (desc) =>
```

Comment:
nit: type for `desc` would be good to have

Reply:
Actually description is being reduced using multiple resources so its hard to define a strict type for it and out of scope for this story. But I have refactored the function to get more specific params with strict types.